judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/SalesModelTest.java

```diff
@@ -83,15 +84,6 @@ public void test() {
 
         assertEquals(1, personList.size());
 
-        Person createdPerson = personDao.create(Person.builder()
-                .withFirstName("Masik")
-                .withLastName("Test")
-                .build());
-
-        assertEquals(Optional.of("Masik"), createdPerson.getFirstName());
-        assertEquals(Optional.of("Test"), createdPerson.getLastName());
-        assertEquals(Optional.of("Masik Test"), createdPerson.getFullName());
-
```

Comment:
Person is abstract, create method is now not generated

Reply:
:)